conf.yaml

```diff
@@ -81,3 +82,67 @@ repository:
                     comment: |
                         {author}'s PR has reached the required number of votes
                         for merging.
+        -
+            name: Check merged PRs for review tag.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: 2016
+            actions:
+                -
+                    action: assert_has_tag
+                    action_value: 'status/review'
```

Comment:
Getting around to implementing this (said long meeting actually required me talking and interacting for 90% of the 5 hours so no dev work got done), just keeping you abreast of the design changes I'm making and why.

`assert_has_tag` is a bad fit for action, since it is essentially just a filter like the rest of conditions. What you're saying here is equivalent to filtering for prs which are lacking the tag, and then executing a comment action as a result.

The philosophy of p4's yaml file isn't fully specified anywhere, but essentially:

> given a group of PRs, find those matching a set of conditions, and execute actions on them like commenting or tagging.

thus, I'm replacing the assert with just another condition, and executing a standard comment action.